### PR TITLE
Wiki Update : Add information about Gradle and ktfmt

### PR DESCRIPTION
**Pull request by KurohanaJuri**

### CODE REVIEW

File: wiki/Home.md [hunk 1]
```diff
@@ -21,26 +22,54 @@ You can find here information about the organization and the workflow of this re
...
+    - Nitpick
+    - Question
+    - Bug
+    - Proposition
+
+## Gradle
+
+Make sure to have gradle tasks in the gradle tab inside Android studio.
+
+![ExepectedGradle](image/ExpectedGradle.png)
```
Comment (alexandrepiveteau) [inline wiki/Home.md:39]:
```suggestion
![ExpectedGradle](image/ExpectedGradle.png)
```

File: wiki/Home.md [hunk 2]
```diff
@@ -21,26 +22,54 @@ You can find here information about the organization and the workflow of this re
...
+
+If you tab isn't like the one on the screen shot, go to `File->Settings...->Experimental`,
```
Comment (alexandrepiveteau) [inline wiki/Home.md:41]:
```suggestion
If your tab isn't like the one on the screen shot, go to `File->Settings...->Experimental`,
```

**Review by alexandrepiveteau — APPROVED** [review]
Thanks !

**Review by alexandrepiveteau — APPROVED** [review]

File: wiki/Home.md [hunk 3]
```diff
@@ -21,26 +22,54 @@ You can find here information about the organization and the workflow of this re
...
+    - Bug
+    - Proposition
+
+## Gradle
+
+Make sure to have gradle tasks in the gradle tab inside Android studio.
+
+![ExpectedGradle](image/ExpectedGradle.png)
+
+If your tab isn't like the one on the screen shot, go to `File->Settings...->Experimental`,
```
Comment (BadrTad) [inline wiki/Home.md:41]:
```suggestion
If your tab isn't like the one on the screenshot, go to `File->Settings...->Experimental`,
```

File: wiki/Home.md [hunk 4]
```diff
@@ -21,26 +22,54 @@ You can find here information about the organization and the workflow of this re
...
+inside `Gradle` section, uncheck `Do not build Gradle task list during Gradle sync`
```
Comment (BadrTad) [inline wiki/Home.md:42]: Proposition: This step alone didn't help to show Gradle tasks tab even after I reloading the IDE. I had to re-build the project. Maybe it's worth to mentioning.

**Review by BadrTad — CHANGES_REQUESTED** [review]

**Review by alexandrepiveteau — APPROVED** [review]

**Review by matt989253 — DISMISSED** [review]

**Review by alexandrepiveteau — APPROVED** [review]

**Review by BadrTad — APPROVED** [review]

**Pull request merged**